core/arch/arm32/sta/sst_km_tests.c

```diff
@@ -0,0 +1,353 @@
+/*
```

Comment:
No problem, I will put the host application to optee_test.git. By the way, the storage test cases in optee_test.git already can cover the good path test of the key manager functionality but not bad path test. The main purpose of the static TA is covering the bad path tests.